src/bundle/Command/AbstractMultiProcessComand.php

```diff
@@ -0,0 +1,356 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\FieldTypeRichText\Command;
+
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\PhpExecutableFinder;
+use Symfony\Component\Process\Process;
+
+abstract class AbstractMultiProcessComand extends Command
+{
+    /** @var \Ibexa\Contracts\Core\Repository\PermissionResolver */
+    protected PermissionResolver $permissionResolver;
+
+    /** @var \Ibexa\Contracts\Core\Repository\UserService */
+    protected UserService $userService;
+
+    /**
+     * @var bool
+     */
+    protected bool $hasProgressBar;
+
+    /**
+     * @var \Symfony\Component\Console\Helper\ProgressBar
+     */
+    protected ProgressBar $progressBar;
+
+    /**
+     * @var \Symfony\Component\Console\Output\OutputInterface
+     */
+    protected OutputInterface $output;
+
+    /**
+     * @var bool
+     */
+    private bool $dryRun;
+
+    /**
+     * @var int
+     */
+    private int $maxProcesses;
+
+    /**
+     * @var \Symfony\Component\Process\Process[]
+     */
+    private $processes;
+
+    /**
+     * @var string
+     */
+    private mixed $user;
+
+    /**
+     * @var int
+     */
+    private int $iterationCount;
+
+    /**
+     * @var string
+     */
+    private string $environment;
+
+    public function __construct(
+        string $name = null,
+        PermissionResolver $permissionResolver,
+        UserService $userService
+    ) {
+        $this->permissionResolver = $permissionResolver;
+        $this->userService = $userService;
+        $this->dryRun = false;
+        $this->processes = [];
+
+        parent::__construct($name);
+    }
+
+    public function configure(): void
+    {
+        $this->
+            addOption(
+                'user',
+                'u',
+                InputOption::VALUE_REQUIRED,
+                'Ibexa DXP username',
+                'admin'
+            )->addOption(
+                'dry-run',
+                null,
+                InputOption::VALUE_NONE,
+                'Run the converter without writing anything to the database'
+            )->addOption(
```

Comment:
When splitting between multiple lines, new method calls should be in separate lines.

This makes it easier to resolve conflicts in git, if they ever occur.
```suggestion
            )
            ->addOption(
                'dry-run',
                null,
                InputOption::VALUE_NONE,
                'Run the converter without writing anything to the database'
            )
            ->addOption(
```